wormhole-connect/src/views/v2/Bridge/index.tsx

```diff
@@ -165,7 +167,11 @@ const Bridge = () => {
     if (validRoutes.length === 0) {
       setSelectedRoute('');
     } else {
-      const autoselectedRoute = route || validRoutes[0].route.name;
+      const preferredRoute = validRoutes.find(
+        (route) => route.route.name === preferredRouteName,
+      );
+      const autoselectedRoute =
+        route ?? validRoutes[0].route.name ?? preferredRoute?.route.name;
```

Comment:
Shouldn't the preferred route come before the `validRoutes[0].route.name`?

Reply:
yes, I've tried to optimize it, and that is why I've moved to the last position, but, you are right, updated